src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -156,12 +156,11 @@ protected function configurePostNormalisation(): void
         );
 
         // Case-insensitive constants handling
-        static $caseInsensitiveConstants = ['null', 'false', 'true'];
         $caseInsensitiveConstantsToEscape = [];
 
         foreach ($constantsToEscape as $constantIndex => $constant) {
             $loweredConstant = strtolower($constant);
-            if (\in_array($loweredConstant, $caseInsensitiveConstants, true)) {
+            if (\in_array($loweredConstant, ['null', 'false', 'true'], true)) {
```

Comment:
`\in_array` is optimized by php when using a static array:
https://3v4l.org/U97pK/vld
You can see in the VLD tab that line 6 and 10 use function calls (`INIT_FCALL` opcode) while line 8 and 12 use optimized `IN_ARRAY` opcodes without a function call.